rust/chains/hyperlane-cosmos/src/merkle_tree_hook.rs

```diff
@@ -104,26 +104,14 @@ impl MerkleTreeHook for CosmosMerkleTreeHook {
     }
 
     /// Gets the current leaf count of the merkle tree
-    #[instrument(level = "debug", err, ret, skip(self))]
```

Comment:
logging the return value was a bit much, but it probably makes sense to still log on errors?